.github/workflows/notify-module-version.yml

```
@@ -0,0 +1,28 @@
+name: Notify MODULE_VERSION changed
+
+on:
+  push:
+    branches:
+    - master
+    - release-*
+    paths:
+    - kernel-modules/MODULE_VERSION
+
+jobs:
+  notify:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Slack notification
+      uses: rtCamp/action-slack-notify@v2
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_ONCALL }}
+        SLACK_CHANNEL: oncall
+        SLACK_COLOR: ${{ job.status }}
+        SLACK_LINK_NAMES: true
+        SLACK_TITLE: MODULE_VERSION has changed!
+        MSG_MINIMAL: commit
+        SLACK_MESSAGE: |
+          @collector-team update downstream with the new version and commit.
+          Create a MR to `rhacs-0.1-rhel-8` adding this information to
+          `upstream_sources.yml` and
+          `distgit/containers/rhacs-drivers-build/container.yaml.in`
```

Improvement suggestion:

```suggestion
          @collector-team you should now update downstream with the new version and commit.
          Create a MR to [rhacs-0.1-rhel-8](https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/tree/rhacs-0.1-rhel-8) adding this `MODULE_VERSION` to
          `upstream_sources.yml` and
          `distgit/containers/rhacs-drivers-build/container.yaml.in`
```